frontend/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
```

Comment:
Ar mums tikrai reikia šių by default sukurtų failų dev aplinkoj?

Reply:
Nereik tikrai :D.
Bet siaip i ateiti ziurint reikes pasidaryt fixtures kurie seedintu db.
https://docs.cypress.io/examples/examples/recipes.html#Seeding-your-Database-in-Node

Reply:
Klausimas. Kaip manai, jei einam per pagrindinį flow: useris prisiregistruoja, atsijungia, useris prisilogina, atlieka pagr. veiksmus, atsilogina. Ar reikia papildomų db duomenų tam, ar galim ir be jų apsieiti?

Reply:
Man atrodo pirmas esminis dalykas butu, kad duombazeje jau yra duomenu is paprasto app naudojimo, todel mes negalim ( be duombazes kontroles ) padaryti logino ar registracijos, nes abu gali unexpectedly failint del to kad jau egzistuoja toks useris :D 

Dar vienas dalykas kuri galvociau, bet as nesu sure apie tai, tai kad _gal_ butu geriau, kad testai butu atominiai, i.e. galetum pratestuoti tik filtravima ar tik transactionu kurima (su seedu pagalba) be to, kad reiketu praeiti keliu transactionu kurima tik tam kad galetum paziuret ar filtravimas veikia.

Reply:
Tai random generator sugeneruotas useris tikrai neturėtų egzistuoti. Pakurtume kelias transactions ir pilnai pakaks filtravimą pratestuoti. Matai, tada klausimas, ar mes e2e, ar integration norim pratestuoti, ar bandyti viską vienu metu.

Reply:
Bet vistiek lieka ta problema, kad istestuoti Filtravima tu turi:

1. Susikurti useri
2. Prisijungti su useriu
3. Susikurti keleta transactionu
4. Tik tada gali filtruoti

Gaunasi taip, kad jeigu registracija/loginas nebeveikia, tai nustoja veikti visi kiti testai :D. Man atrodo reiketu vengtu tokio strong coupling ir tureti mazesnio scope testus. I.e. turint tokius transactionus (fixture) filtravimas turetu veikti taip